internal/docs/doc-login.md

```diff
@@ -61,6 +82,9 @@ against auth0, you need to change it to "client_credentials".
 
 There is (currently) no caching of tokens, so if you use Auth0, be advised that this can incur additional costs.
 
+### Token login profile
+
+We don't recommend you to use this type, but sometimes you just want to test something so it is provided.
```

Comment:
Again I think we should be neutral here in terms of capabilities and have opinions else where.